CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog
 
+### Next version
+- Support AWS China partition under amazonaws.com.cn for EC2 and S3 endpoints.
```

Comment:
We no longer use this, we use the releases page for changelogs so you should revert this

Reply:
My apologies, I've reverted in new commit.